### ISSUE TITLE
How to label images with lots of objects correctly?

### ISSUE THREAD

**Issue by ashgard** [pjreddie/darknet#2029]
Suppose i want to train a detector that detects flower bulbs in an image of an orchid. An orchid can contain lots of bulbs (>20). That means my training images also contain that amount of objects (bulbs). Now my question. Should i label all bulbs in the image? My concern/question is, suppose i do not label all of them, is the part of the image that is not labelled used for "negative" samples? Or are only images that do not contain any object used for "negative" samples?

**Comment by nick-hunter** [comment]
Yes I would label all of them. I believe your concern about negative samples is correct, but I would recommend reading the YOLO papers. Section 2.2 of the first YOLO paper talks about confidence scores for boxes that don't contain objects.
